extension/content.js

```diff
@@ -179,10 +179,13 @@ function addPatchDiffLinks() {
 }
 
 function removeDiffSigns() {
-	$('.blob-code-deletion .blob-code-inner, .blob-code-addition .blob-code-inner').each((index, element) => {
-		const textNode = element.childNodes[0];
-		textNode.nodeValue = textNode.nodeValue.replace(/^[+-]/, ' ');
-	});
+	$('.blob-code-addition, .blob-code-deletion')
+		.find('.blob-code-inner:not(.refined-github-diff-signs)')
+		.each((index, element) => {
+			const $element = $(element);
+			$element.html($element.html().slice(1));
+			$element.addClass('refined-github-diff-signs');
+		});
```

Comment:
I noticed now that this change makes some diffs unaligned.

For example: https://github.com/sindresorhus/globby/pull/40/files#diff-354f30a63fb0907d4ad57269548329e3

<img width="149" alt="screen shot 2017-05-02 at 11 41 38" src="https://cloud.githubusercontent.com/assets/170270/25604258/57f64064-2f2c-11e7-9f77-90b84004bcf6.png">